networked_controller.cpp

```diff
@@ -639,11 +721,9 @@ void ServerController::process(real_t p_delta) {
 
 	node->get_inputs_buffer_mut().begin_read();
 	node->get_inputs_buffer_mut().seek(METADATA_SIZE);
-	GDVIRTUAL_CALL_PTR(
-			node,
-			_controller_process,
+	node->native_controller_process(
```

Comment:
why NetworkedController instance is called "node"? It was not obvious for me what this member is representing (and if some type casting is not coming into play here)